app/controllers/providers/proceedings_sca/child_subjects_controller.rb

```diff
@@ -0,0 +1,46 @@
+module Providers
+  module ProceedingsSCA
+    class ChildSubjectsController < ProviderBaseController
+      prefix_step_with :proceedings_sca
+
+      def show
+        @proceeding = legal_aid_application.proceedings.last
+        form
+      end
+
+      def update
+        return continue_or_draft if draft_selected?
+
+        @proceeding = legal_aid_application.proceedings.last
+        form
```

Comment:
I'm not sure you need the extra form call, if form.valid? on the next line will instantiate it